### PR TITLE
LCA ontology

### PR DESCRIPTION
**Pull request by JulKaltenegger**

### CODE REVIEW

File: bmp/.htaccess [hunk 1]
```diff
@@ -37,7 +37,7 @@ RewriteRule ^$ https://julkaltenegger.github.io/bmp/ontology.nt [R=303,L]
 RewriteCond %{HTTP_ACCEPT} text/turtle [OR]
 RewriteCond %{HTTP_ACCEPT} text/\* [OR]
 RewriteCond %{HTTP_ACCEPT} \*/turtle 
-RewriteRule ^$ https://julkaltenegger.github.io/bmp/ontology.ttl [R=303,L]
+RewriteRule ^$ https://julkaltenegger.github.io/bmp/ontology.html [R=303,L]
```
Comment (davidlehn) [inline bmp/.htaccess:40]: Looks like this got in by mistake?

**Comment by JulKaltenegger** [comment]
Yes. I wanted to add the lca ontology only.
Working from two laptops (transition), hence the overlap.
Did you receive the lca pull request?

**Comment by dgarijo** [comment]
Yes, but you are also modifying the other file (in `bmp`). Please amend that edit if it was not intended

**Comment by JulKaltenegger** [comment]
i think it should be fine now?

**Pull request merged**